frontend/src/components/graphs/TimeSeriesLineChart.js

```diff
@@ -32,19 +38,22 @@ function TimeSeriesLineChart(props) {
     colors: ['#f00', '#ffd700', '#c71585', '#0f0', '#117519', '#00f', '#0ff'],
     stroke: {
       curve: 'straight',
-    },
+    }
   };
 
   return (
-    <div id="chart">
-      <ReactApexChart
-        options={options}
-        series={data.series}
-        type="line"
-        height={400}
-        width="100%"
-      />
-    </div>
+    <>
+      <div id="chart" >
+        <ReactApexChart
+          options={options}
+          series={data.series}
+          type="line"
+          height={400}
+          width="100%"
+        />
+      </div>
+      <CopyGraphButton key={animationEnded} chartId='line-chart' animationEnded={animationEnded}/>
```

Comment:
De copy to clipboard werkt niet op mobile misschien hier {!isMobile && <>} van maken?